x-pack/plugins/apm/public/components/app/ServiceMap/Popover/Contents.tsx

```diff
@@ -51,14 +77,53 @@ const FlexColumnGroup = (props: {
 const FlexColumnItem = (props: { children: React.ReactNode }) =>
   isIE11 ? <div {...props} /> : <EuiFlexItem {...props} />;
 
+const ANOMALY_DETECTION_TITLE = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverTitle',
+  { defaultMessage: 'Anomaly Detection' }
+);
+
+const ANOMALY_DETECTION_INFO = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverInfo',
+  {
+    defaultMessage:
+      'Display the health of your service by enabling the anomaly detection feature in Machine Learning.'
+  }
+);
+
+const ANOMALY_DETECTION_SCORE_METRIC = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverScoreMetric',
+  { defaultMessage: 'Score (max.)' }
+);
+
+const ANOMALY_DETECTION_LINK = i18n.translate(
+  'xpack.apm.serviceMap.anomalyDetectionPopoverLink',
+  { defaultMessage: 'View anomalies' }
+);
+
 export function Contents({
   selectedNodeData,
   isService,
   label,
   onFocusClick,
   selectedNodeServiceName
 }: ContentsProps) {
-  const frameworkName = selectedNodeData[SERVICE_FRAMEWORK_NAME];
```

Comment:
Is the intention to add framework name in 7.9? If not, then also remove `SERVICE_FRAMEWORK_NAME` from the es queries and responses.

Reply:
Not at this time, no - so pls remove